test/exchange-files.js

```diff
@@ -162,10 +173,16 @@ describe('exchange files', function () {
               number: num
             })
 
-            const { cid } = await last(daemon1.api.addAll(globSource(dir, '**/*'), {
+            const res = await last(daemon1.api.addAll(globSource(dir, '**/*'), {
               wrapWithDirectory: true
             }))
 
+            if (res == null) {
+              throw new Error('Nothing added')
```

Comment:
I'm not sure if this error is helpful for devs running tests or just for getting the test to function correctly. If the former, can we provide more context for the error?